contrib/build-windows.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh
+set -e
+
+# install deps
+if [ ! -f /usr/share/mingw/toolchain-mingw64.meson ]; then
+    sudo dnf install -y \
+        ca-certificates \
+        diffutils \
+        gcab \
+        gcc \
+        gettext \
+        glib2-devel \
+        gnutls-utils \
+        https://kojipkgs.fedoraproject.org//work/tasks/3617/86733617/msitools-0.101.32-4.fc36.x86_64.rpm \
+        jq \
+        mingw64-brotli \
+        mingw64-curl \
+        mingw64-gcc \
+        mingw64-gettext \
+        mingw64-glib2 \
+        mingw64-gnutls \
+        mingw64-json-glib \
+        mingw64-libarchive \
+        mingw64-libusbx \
+        mingw64-pkg-config \
+        mingw64-sqlite \
+        tree \
+        wine
+fi
+
+root=$(pwd)
+export DESTDIR=${root}/dist
+build=$root/build-win32
+mkdir -p "$build" && cd "$build"
+
+# try to keep this and ../contrib/ci/build_windows.sh in sync as much as makes sense
```

Comment:
How about just source them from one another rather than have to keep in sync?

Reply:
I did ponder putting them in `contrib/ci/dependencies.xml` -- e.g.

      <package variant="mingw64">mingw64-curl</package>

insane?

Reply:
Not insane, makes perfect sense!

Reply:
Ya,concur, it kinda works.

Reply:
Drop the manual install steps now from circleci too.

Reply:
Drop the manual install steps now from circleci too.

Reply:
On my phone maybe i missed it. I was meaning  in .circleci/config.yaml